### ISSUE TITLE
Uncaught ReferenceError: DisasterRecoveryEvents is not defined

### ISSUE THREAD

**Issue by yu-shiba** [amazon-connect/amazon-connect-streams#385]
After updating to 1.6.4, the following error occurred and it did not work.
```
amazon-connect-1.6.4.js:22744 Uncaught ReferenceError: DisasterRecoveryEvents is not defined
    at amazon-connect-1.6.4.js:22744
    at amazon-connect-1.6.4.js:22745
(anonymous) @ amazon-connect-1.6.4.js:22744
(anonymous) @ amazon-connect-1.6.4.js:22745
```

**Comment by configlab** [comment]
The missing code is the following enumeration, which has been verified. 
----------------
  DisasterRecoveryEvents=connect.makeEnum([
    'SUPPRESS',
    'FORCE_OFFLINE',
    'SET_OFFLINE',
    'INIT_DISASTER_RECOVERY'
]);
---------------
[config.net.cn](http://config.net.cn) , focusing on the installation and configuration of various development environments

**Comment by yu-shiba** [comment]
The problem has been resolved in 1.6.5 and is now closed.
https://github.com/amazon-connect/amazon-connect-streams/releases/tag/1.6.5
